tailwind.config.mjs

```diff
@@ -0,0 +1,25 @@
+export default {
```

Comment:
In real life, we wouldn't define most or any of this config here; we'd import and (maybe) extend the preset config provided by `frontend-shared`.